apps/docs/src/pages/colors.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: Colors
+description: Sample Colors documentation
+layout: ../layouts/MainLayout.astro
+---
+
+import { ColorWrapper, ColorsWrapper } from '@selleo/core/src/Colors';
+import { H2 } from '@selleo/core/src/Headers';
+import Preview from '../components/Preview.astro';
+import Description from '../components/Description.astro';
+
+<Description>
+  Below you can find Colors components styled with tailwindcss.
+</Description>
+
+<H2>Brand Colors</H2>
+
+<Preview>
+  <ColorWrapper
+    size="big"
+    class="bg-brand-primary-500 [&_p]:text-white [&_h4]:text-white"
+    colorInfo={{
+      title: 'primary.500',
+      description: 'HEX #FF6D2A | hsl(19, 100, 58) | Pantone 165 C',
+    }}
+  />
+</Preview>
+
+<Preview>
+  <ColorsWrapper itemsLength={3}>
+    <>
+      <ColorWrapper
+        colorInfo={{
+          title: 'primary.600',
+          description: 'HEX #E25718 | hsl(19, 81, 49)',
+        }}
+        size="small"
+        class="bg-brand-primary-600 [&_p]:text-white [&_h4]:text-white"
+      />
+      <ColorWrapper
+        colorInfo={{
+          title: 'primary.400',
+          description: 'HEX #FF905D | hsl(19, 100, 68)',
+        }}
+        size="small"
+        class="bg-brand-primary-400 &_p]:text-white [&_h4]:text-white"
```

Comment:
please create an object on top containing all the colors info and map through them in render. Easier management

Reply:
Yes, definitely it'll be easier in the future 😄